src/main/java/seedu/address/model/person/Block.java

```diff
@@ -13,6 +16,7 @@
     public static final String MESSAGE_CONSTRAINTS =
             "Block should only contain alphabets";
     public static final String VALIDATION_REGEX = "[a-zA-Z]";
+    private static List<String> blockPref = Arrays.asList("A", "B", "C", "D");
```

Comment:
Should `blockPref` be derived from `UserPrefs#getBlockSettings()`?

Reply:
This is the default value which be overridden by `setBlockPref` , its needed for testing since all the test cases do not read from the preference file

Reply:
Okay, noted.